app/services/unsplash_service.rb

```diff
@@ -0,0 +1,16 @@
+class UnsplashService
+  def get_image(location)
+    resp = conn.get do |req|
+      req.params['query'] = location.split(',')[0]
```

Comment:
Only searching for the city because the search with the comma and state were giving me weird results.